### ISSUE TITLE
Zeplin in RFC

### ISSUE THREAD

**Issue by Deathweeler** [wow-mania/Redemption#260]
![wowscrnshot_080315_044910](https://cloud.githubusercontent.com/assets/13618327/9030592/9f9662aa-399c-11e5-95fc-dd9db6c4676b.jpg)
Hello, I've just entered into RFC to do my Paladin Quest. When I entered, There was a Zeplin in the way... I cannot get out because it's blocking the exit. I know I could HearthStone but I need to go to the barrens after..

**Comment by wow-mania** [comment]
Fixed.

**Comment by Rushor** [comment]
how do you fxed thos old problem? or do you closed the issue without further ivestidation?

**Comment by fvicaria** [comment]
What problem the zepelin?

**Comment by Rushor** [comment]
yeah this strange issue that the zepelin appears in the instance :D

**Comment by fvicaria** [comment]
I dismissed it as an X Files. I managed to catch it only once. After I manually deleted it I have not seeing it again. I can't find any reference in code or in objects table.
